instance/models/mixins/openedx_config.py

```diff
@@ -9,6 +9,7 @@
 from django.conf import settings
```

Comment:
@viadanna, can you add unit tests for the changes made in this PR?

Reply:
Sure :+1: